src/main/java/where2eat.ump

```diff
@@ -0,0 +1,68 @@
+namespace ca.mcgill.ecse.where2eat.model;
+
+enum Category {Chinese, Indian, Portuguese}
+
+class Where2Eat {
+  1 <@>- * User users;
+  1 <@>- * Login loginIDs;
+  1 <@>- * Group groups;
+  1 <@>- * Restaurant restaurants;
+  1 <@>- * Location locations;
+  1 <@>- * Address addresses;
+  1 <@>- * PhoneNumber phoneNumbers;
+}
+
+class User
+{
+  String firstName;
+  String lastName;
+  String userID;
+  Category category;
```

Comment:
This should be `Set<Category> categories` IMO. We could opt for an array but I think a Set lets us be extensible - we just enforce the limit on size manually.